tools/clang/test/HLSL/ShaderOpArith.xml

```diff
@@ -3655,6 +3691,17 @@ void MSMain(uint GID : SV_GroupIndex,
 
     ROOT_SIG
     float4 PSMain(PSInput input) : SV_TARGET {
+    
+        int x = (int)input.position.x;
+        int y = (int)input.position.y;
+        int ix = x+y;
```

Comment:
Double check that this doesn't end up being zero for all/most of your pixels based on the incoming range for position x and y.